src/views/FormView.vue

```diff
@@ -0,0 +1,392 @@
+<script>
+import { reactive, ref, computed } from "vue";
+import { useCardiomyopathyDataStore } from "@/stores/CardiomyopathyDataStore";
+import {
+  MDBRadio,
+  MDBInput,
+  MDBIcon,
+  MDBBtnClose,
+} from "mdb-vue-ui-kit";
+import FormInput from "@/components/FormInput.vue";
+import AjaxButton from "@/components/AjaxButton.vue"; 
+
+export default {
+  components: {
+    MDBRadio,
+    MDBInput,
+    MDBIcon,
+    MDBBtnClose,
+    FormInput,
+    AjaxButton, 
+  },
+
+  setup() {
+    const cardiomyopathyDataStore = useCardiomyopathyDataStore();
+    const genes = ref([]);
+    const syncing = ref(false); 
+
+    const newGeneName = ref(null);
+    const newGeneComputedName = computed({
+      get: () => newGeneName.value,
+      set: (newName) => {
+        newGeneName.value = newName.toUpperCase();
+      },
+    });
+
+    function addGene() {
+      if (newGeneName.value) {
+        genes.value.push(newGeneName.value);
+        newGeneName.value = null;
+      }
+    }
+
+    const form = reactive({
+      cardiomyopathyType: "",
+      ledv: "",
+      redv: "",
+      lesv: "",
+      resv: "",
+      lvef: "",
+      rvef: "",
+      lvmass: "",
+      rvmass: "",
+      lsv: "",
+      rsv: "",
+      scar: "",
+      gender: "",
+      mriAge: "",
+      apicalHCM: "",
+      suddenCardiacDeath: "",
+      hypertension: "",
+      diabetes: "",
+      myectomy: "",
+    });
+    const submitData = () => {
```

Comment:
Change to suggestion
```suggestion
    funciton submitData() {
```